daemon/cmd/identity.go

```diff
@@ -86,3 +85,13 @@ type CachingIdentityAllocator interface {
 
 	Close()
 }
+
+func (d *Daemon) AddIdentity(id *identity.Identity) {
+	d.idmgr.Add(id)
+}
+func (d *Daemon) RemoveIdentity(id *identity.Identity) {
+	d.idmgr.Remove(id)
+}
+func (d *Daemon) SwapOldNewIdentity(old, new *identity.Identity) {
```

Comment:
nit: Any reason to keep the name inconsistent here?
Can it be `RemoveOldAddNewIdentity`?

Reply:
Renamed